via/templates/view_video.html.jinja2

```diff
@@ -16,7 +16,16 @@
       {
         "client_config": {{ client_config | tojson }},
         "client_src": "{{ client_embed_url }}",
+        "player": {{ player | tojson }},
```

Comment:
Why is the `tojson` filter needed here? Isn't `player` a string?

Reply:
`tojson` will wrap the string in quotes. `"{{ player }}"` would have worked equally well.